pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, fetchFromGitHub
+, substituteAll
+, buildNpmPackage
+, makeDesktopItem
+, copyDesktopItems
+, vencord
+, electron
+, pipewire
+, libicns
+}:
+let
+  libPath = lib.makeLibraryPath [ pipewire ];
+in
+buildNpmPackage rec {
+  pname = "vesktop";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "Vencord";
+    repo = "Vesktop";
+    rev = "v${version}";
+    sha256 = "sha256-Njs3tACxUyRolYUtS/q2lITIQnUBFXVXWZEfQ66HpPM=";
+  };
```

Comment:
```suggestion
buildNpmPackage (finalAttrs: {
  pname = "vesktop";
  version = "0.3.3";

  src = fetchFromGitHub {
    owner = "Vencord";
    repo = "Vesktop";
    rev = "v${finalAttrs.version}";
    sha256 = "sha256-Njs3tACxUyRolYUtS/q2lITIQnUBFXVXWZEfQ66HpPM=";
  };
```

Reply:
huh?

Reply:
I just remembered that this doesn't work with buildNpmPackage (yet). Disregard this!